scout/server/blueprints/variant/templates/variant/cancer-variant.html

```diff
@@ -0,0 +1,363 @@
+      {% if variant.rank_score_results %}
+        <div class="col-12">{{ rankscore_panel(variant) }}</div>
+      {% endif %}
+    </div>
+
+    {% if variant.compounds %}
+      <div class="row">
+        <div class="col-12">{{ compounds_panel(institute, case, variant) }}</div>
+      </div>
+    {% endif %}
+    <div class="row">
+    </div>
+    <div class="row">
+      <div class="col-12">
+        <div class="card ">
+          <nav>
+            <div class="nav nav-tabs" id="nav-tab" role="tablist">
+              <a class="nav-item nav-link active" id="nav-genes-tab" data-toggle="tab" href="#nav-genes" role="tab" aria-controls="nav-genes" aria-selected="true">Genes</a>
+              <a class="nav-item nav-link" id="nav-transcripts-tab" data-toggle="tab" href="#nav-transcripts" role="tab" aria-controls="nav-transcripts" aria-selected="false">Transcripts</a>
+              <a class="nav-item nav-link" id="nav-proteins-tab" data-toggle="tab" href="#nav-proteins" role="tab" aria-controls="nav-proteins" aria-selected="false">Proteins</a>
+            </div>
+
+          </nav>
+          <div class="tab-content" id="nav-tabContent">
+            <div class="tab-pane fade show active" id="nav-genes" role="tabpanel" aria-labelledby="nav-genes-tab">{{ genes_panel(variant) }}</div>
+            <div class="tab-pane fade" id="nav-transcripts" role="tabpanel" aria-labelledby="nav-transcripts-tab">{{ transcripts_panel(variant) }}</div>
+            <div class="tab-pane fade" id="nav-proteins" role="tabpanel" aria-labelledby="nav-proteins-tab">{{ proteins_panel(variant) }}</div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-12">{{ external_links(case, variant) }}</div>
+    </div>
+    <div class="row">
+      <div class="col-12">{{ activity_panel(events|reverse) }}</div>
+    </div>
+    {% if config['MAIL_USERNAME'] %}
+      {# Email setting must be setup #}
+      {{ modal_sanger(case, variant, current_user) }}
+    {% endif %}
+
+    {% if config['MAIL_USERNAME'] %}
+      {# Email setting must be setup #}
+      {{ modal_cancel_sanger(case, variant, current_user) }}
+    {% endif %}
+</div>
+{{ modal_causative(case, institute, variant) }}
+{% endblock %}
+
+{% macro panel_quality() %}
+  <div class="card panel-default">
+    <div class="panel-heading">Quality</div>
+    <ul class="list-group list-group-flush">
+      <li class="list-group-item">QUAL:
+        <span class="font-weight-bold">
+          {% if variant.quality %}
+            {{ variant.quality|int }}
+          {% else %}
+            -
+          {% endif %}
+        </span>
+        <li class="list-group-item">{{ mappability(variant) }} </li>
+    </ul>
+  </div>
+{% endmacro %}
+
+
+{% macro panel_comment() %}
+  <div class="card panel-default">
+    {{ comments_panel(institute, case, current_user, variant.comments, variant_id=variant._id) }}
```

Comment:
Comments panel looks tiny on the view, but there is a lot of empty space on its left side. And also the transcripts overview (line down, left) could use some extra space. What about moving the GT call panel up, on the side of comments and make them each col-6 (splice the row 50-50) So you are free to expand the transcripts below?

![image](https://user-images.githubusercontent.com/28093618/115831569-0ed2e180-a412-11eb-96f3-9c446f8b6d71.png)